### PR TITLE
Default Mesh by Drone Team

### PR DESCRIPTION
**Pull request by sergei-petroff**
This Pull Request contains a big amount of changes made by Drone Project Team. 
The main additional feature is the Default Mesh capability. Default Mesh settings are read from the env variables provided to the docker container.
More details:
- Default Mesh
- ROS baseimage: fog-ros-sdk:v3.2.0
- Removed gateway autoselect capability

### CODE REVIEW

File: common/tools/field_test_log_plotter/ftl_plotter.py [hunk 1]
```diff
@@ -400,6 +400,17 @@ def __convert_units(self):
...
+        # Convert throughput values
+        if self.throughput_units == 'Kb':
+            tp_divider = 1000
+        elif self.throughput_units == 'Mb':
+            tp_divider = 1000000
+        if tp_divider > 1:
+            # Converting only actual values but not changing column name.
+            self.df['RX throughput [Bits/s]'] = self.df['RX throughput [Bits/s]'] / tp_divider
+            self.df['TX throughput [Bits/s]'] = self.df['TX throughput [Bits/s]'] / tp_divider
+
```
Comment (joenpera) [inline common/tools/field_test_log_plotter/ftl_plotter.py:413]: Why to add second time same code block?

File: common/tools/field_test_log_player/README.md [hunk 2]
```diff
@@ -47,4 +47,4 @@ Active keys in plotter view:
 
     spacebar    - pause/resume, zooming possible during pause
     +           - skip next 30seconds in log file
-    -           - rewind 30seconds in log file
\ No newline at end of file
+    -           - rewind 30seconds in log file
```
Comment (joenpera) [inline common/tools/field_test_log_player/README.md:50]: Why this change?

File: common/scripts/default-mesh.sh [hunk 3]
```diff
@@ -0,0 +1,82 @@
+#!/bin/bash -eu
+
+# Check if proper default mesh class is set
+if [[ "$MESH_CLASS" != "gs" && "$MESH_CLASS" != "edge" ]]; then
+  echo "Wrong mesh class, check container env parameters"
+  exit 1
+else
+  echo "Mesh class: " $MESH_CLASS
```
Comment (joenpera) [inline common/scripts/default-mesh.sh:8]: Lot of double quotes missing around the variables (to avoid word splitting).  Please check the file with shellcheck tool.

File: common/tools/field_test_log_plotter/ftl_plotter.py (hunk 1 above)
Comment (sergei-petroff) [reply common/tools/field_test_log_plotter/ftl_plotter.py]: You are right, this is merge glitch. Duplicate code removed.

File: common/tools/field_test_log_player/README.md (hunk 2 above)
Comment (sergei-petroff) [reply common/tools/field_test_log_player/README.md]: Correct, this is unnecessary line. Removed.

File: common/scripts/default-mesh.sh (hunk 3 above)
Comment (sergei-petroff) [reply common/scripts/default-mesh.sh]: Done according to the shellcheck recommendations.

**Review by joenpera — APPROVED** [review]
:gem:

**Review by saauvine — APPROVED** [review]
